maths/find_min.py

```diff
@@ -1,17 +1,21 @@
-"""Find Minimum Number in a List."""
+def find_min(nums):
+    """
+    Find Minimum Number in a List
+    :param nums: contains elements
+    :return: max number in list
 
+    >>> find_min([1, 3, 5, 7, 9]) == 1
```

Comment:
Let's use the same tests data as find_max.py uses.

Reply:
Updated. Please review again